src/features/author/AuthorPage/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import Header from "../../common/Header";
-import Section from "../../common/Section";
-import Container from "../../common/Container";
+import Container from '../../../common/Container';
```

Comment:
a good way to manage those imports and make them look less annoying is to implement aliases as they are widely used among react projects - take a look here (https://dev.to/rfornal/create-path-aliases-in-react-2oid)